server/apps/distributed-app/docs/modules/ROOT/pages/configure/queue.adoc

```diff
@@ -0,0 +1,19 @@
+= Distributed James Server &mdash; queue.properties
+:navtitle: queue.properties
+
+This configuration helps you configure mail queue you want to select.
+
+== Queue Configuration
+
+.queue.properties content
+|===
+| Property name | explanation
+
+| queue.choice
+| Mail queue can be implemented by many different type of message brokers: Pulsar, RabbitMQ,.... This property will choose which mail queue you want, defaulting to RABBITMQ
+|===
+
+`queue.choice` supports the following options:
```

Comment:
Rethink about the property name, I think maybe the name  e.g. `mail.queue.choice` would be more clear.

Because we have other queues e.g. event queue and task queue, the current property seems easy to misunderstand. I would tend to understand all queue types would be solely using Pulsar for example.